snippets/legv8.json

```diff
@@ -0,0 +1,185 @@
+{
+  "set register to": {
+    "prefix": "sreg",
+    "body": ["ADD ${1:dest reg}, XZR, $2", "$3"]
+  },
+  "increment register": {
+    "prefix": "increg",
+    "body": ["ADDI ${1:dest reg}, ${1}, #1", "$2"]
+  },
+  "decrement register": {
+    "prefix": "decreg",
+    "body": ["SUBI ${1:dest reg}, ${1}, #1", "$2"]
```

Comment:
Why do your "body" arrays have the parameter at the end? To put the cursor to the next line?

Reply:
exactly, it helps maintain the flow a bit better